### PR TITLE
BUG: fix and add missing codes to the Slicer terminology file

### LINKED ISSUE #388: Missing items in Slicer general anatomy
I've tried to use [SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json](https://github.com/QIICR/dcmqi/blob/master/doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json) terminology file but found that very basic structures are missing. So, I compared the json file to [Slicer anatomy lookup table](https://www.slicer.org/w/index.php/Slicer3:2010_GenericAnatomyColors#Lookup_table) and found a couple of missed items and other small inconsistencies.

Present in DICOM context file (https://github.com/QIICR/dcmqi/blob/master/doc/segContexts/SegmentationCategoryTypeModifier-DICOM-Master.json) but for some reason missing from SlicerGeneralAnatomy context file (https://github.com/QIICR/dcmqi/blob/master/doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json):
- [x] gray matter
- [x] liver
- [x] maxilla
- [x] midbrain
- [x] pleura
- [x] ribs
- [x] spleen
- [x] telencephalon
- [x] vagus nerve
- [x] white matter

Other issues:
- [x] diencephalon: 3dSlicerLabel is assigned to the "right" modifier instead of the main item
https://github.com/QIICR/dcmqi/blob/c3b40ac82ebcf760325884c6558e1aacbe622133/doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json#L1046
- [x]  occipital bone: missing from DICOM file, too, which is odd, because the frontal, parietal, temporal, sphenoid, nasal bones are all listed (https://en.wiktionary.org/wiki/occipital_bone)
- [x]  vomer bone: missing from DICOM file, too, which is odd, because the frontal, parietal, temporal, sphenoid, nasal bones are all listed (https://en.wikipedia.org/wiki/Vomer)

### PR DESCRIPTION
**Pull request by fedorov**
This should resolve #388

### CODE REVIEW

File: doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json [hunk 1]
```diff
@@ -1034,22 +1310,10 @@
...
-                "cid": "244",
-                "CodingSchemeDesignator": "SRT",
-                "3dSlicerLabel": "diencephalon",
-                "CodeValue": "G-A100",
-                "UMLSConceptUID": "C0205090",
-                "CodeMeaning": "Right",
-                "contextGroupName": "Laterality",
-                "SNOMEDCTConceptID": "24028007"
-              }
+            "recommendedDisplayRGBValue": [
```
Comment (lassoan) [inline doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json]: Please add  3dSlicerLabel and 3dSlicerIntegerLabel

File: doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json [hunk 2]
```diff
@@ -2759,6 +3023,60 @@
             ],
             "SNOMEDCTConceptID": "24924006"
           },
+          {
+            "contextGroupName": "Craniofacial Anatomic Regions",
+            "cid": "4028",
+            "CodeMeaning": "Occipital bone",
```
Comment (lassoan) [inline doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json]: add 3dSlicerLabel and 3dSlicerIntegerLabel for the entire occipital bone (https://www.slicer.org/wiki/Slicer3:2010_GenericAnatomyColors -> 151)

File: doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json [hunk 3]
```diff
@@ -2759,6 +3023,60 @@
...
+            "Modifier": [
+              {
+                "recommendedDisplayRGBValue": [
+                  182,
+                  166,
+                  110
+                ],
+                "cid": "244",
+                "CodingSchemeDesignator": "SRT",
+                "3dSlicerLabel": "right occipital bone",
```
Comment (lassoan) [inline doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json]: add 3dSlicerIntegerLabel

File: doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json [hunk 4]
```diff
@@ -2759,6 +3023,60 @@
...
+              },
+              {
+                "recommendedDisplayRGBValue": [
+                  229,
+                  204,
+                  109
+                ],
+                "cid": "244",
+                "CodingSchemeDesignator": "SRT",
+                "3dSlicerLabel": "left occipital bone",
```
Comment (lassoan) [inline doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json]: add 3dSlicerIntegerLabel

File: doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json [hunk 5]
```diff
@@ -2759,6 +3023,60 @@
...
+          },
+          {
+            "recommendedDisplayRGBValue": [
+              255,
+              237,
+              145
+            ],
+            "cid": "4028",
+            "CodingSchemeDesignator": "SRT",
+            "3dSlicerLabel": "vomer bone",
```
Comment (lassoan) [inline doc/segContexts/SegmentationCategoryTypeModifier-SlicerGeneralAnatomy.json]: add 3dSlicerIntegerLabel

**Comment by fedorov** [comment]
Thank you, I completely missed this as I was pasting items from the DICOM context file.

Now that I am reviewing it on this issue, I see only 11 structures have `3dSlicerIntegerLabel` initialized. I think I should write a script to populate those items for all structures.

**Comment by lassoan** [comment]
@fedorov Are you going to send a pull request to update the files bundled with Slicer? https://github.com/Slicer/Slicer/tree/master/Modules/Loadable/Terminologies/Resources

**Comment by fedorov** [comment]
Indeed, I should do it. But I am not sure I will get to it today or tomorrow due to New Year.

**Comment by lassoan** [comment]
Thank you! It's not urgent, many people are on vacation these days anyway.

**Pull request merged**